page-process-list/public/locales/pt-BR.json

```diff
@@ -6,18 +6,16 @@
   "Version": "Versão",
   "Categories": "Categorias",
   "Description": "Descrição",
-  "Start a new case": "Iniciar um novo caso",
   "Action": "Ação",
-  "AllProcesses": "TodosProcessos",
+  "Start a new case": "Iniciar um novo caso",
+  "AllCategories": "All categories",
```

Comment:
We lost the "TodosProcessos" translation

Reply:
It seems normal as it is no longer in en.js